tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -842,11 +842,48 @@ def _restore_collections(dest_graph, src_meta_graph_def, collection_keys):
     # Ignore other meta graphs from the input SavedModel.
     saved_model_builder.save()
 
-
 def _get_resource_handle(name, device):
   with ops.device(device):
-    return gen_trt_ops.create_trt_resource_handle(resource_name=name)
+    return gen_trt_ops.create_trt_resource_handle(resource_name=name) 
 
+def _remove_native_segments(input_func):
+  """ Remove native segments from the input TF-TRT Converted Function.
+  Args:
+    input_func: provide the concrete function with native segment nodes. 
+     The transformed output func will not contain any native segment nodes.
+     All the TRTEngineOp references will be modified to a string "deleted"
+     to automatically identify that the associated native segment is deleted.
```

Comment:
This is not true anymore, right? It seems that you are deleting the `segment_func` attribute.  
IIUC, this means that the `segment_func` attribute will have a default value default value, which has an empty name.

Reply:
That is correct, I fixed it.